v2.1/export.md

```diff
@@ -94,7 +94,7 @@ Convert SQL *NULL* values so they match the specified string.
 		</tr>
 		<tr>
 			<td><strong>Value</strong></td>
-			<td>The string that should be used to represent <em>NULL</em> values</td>
+			<td>The string that should be used to represent <em>NULL</em> values. It is important to pick `nullas` values unlikely to appear in the exported data to avoid collusions.</td>
```

Comment:
I'm deeply worried about collusion, but right here I think it is collisions I'm more worried about. 

Also, I'd probably say we don't want to soften it with "unlikely". It is important to pick a `nullas` values that does not appear in the exported data if is important to distinguish it later.

Reply:
nit: `nullas` should be formatted as `<code> </code>`